### PR TITLE
add some new copy to commitments page and some number boxes

### PR DESCRIPTION
**Pull request by emilyville**
I couldn't find the final copy for this one - I saw the link to https://github.com/department-of-veterans-affairs/vets-website/pull/1600 but this was confusing - so I just grabbed the first paragraph. 

We should revisit this guy soon, but all in all should work and is an improvement!!

screencapture-localhost-3000-commitments-1459405424458

### CODE REVIEW

**Comment by gnakm** [comment]
This is great!

File: app/views/employers/commitments.html.erb [hunk 1]
```diff
@@ -17,72 +17,96 @@
   <div class="section one">
     <% unless user_signed_in? && current_user.is_employer? %>
       <div class="row">
-          <div class="small-12 columns">
-          <p>Employers every day are committing to hire Veterans &mdash; you should too! Check out the list below to see what employers have already made a commitment and then <%= link_to "create or update your profile", new_user_session_path %> to signal your commitment.</p>
+        <div class="small-12 medium-8 columns">
+          <div class="primary">
+            <p>Veterans and military family members have consistently proven themselves to be the most reliable, independent, and hardworking team members. Additionally, Veterans have soft skills such as problem solving and creativity that can benefit a company’s bottom line. By signing up and <a href="/employment/commitments">making a commitment</a>, your company will stand out to candidates eager to enter the civilian workforce.</p>
```
Comment (ayaleloehr) [inline app/views/employers/commitments.html.erb]: We're already on the commitments page, so having a link here does not seem helpful (will just be a link to the page you're already on)

**Comment by ayaleloehr** [comment]
One nit, otherwise LGTM.  @emilyville if you remove that link, feel free to merge.

**Comment by ayaleloehr** [comment]
Thanks!  LGTM

**Pull request merged**